src/translations/screens/subscreens/MobilityTexts.ts

```diff
@@ -12,7 +12,7 @@ export const MobilityTexts = {
       case FormFactor.Scooter:
         return _('Sparkesykkel', 'Electric scooter', 'Elektrisk sparkesykkel');
       case FormFactor.Bicycle:
-        return _('Sykkel', 'Bicycle', 'Sykkel');
+        return _('Bysykkel', 'Bicycle', 'Bysykkel');
```

Comment:
True, should I use `Sykkel` instead? since the Figma uses `Bysykkel` on the example 
![image](https://github.com/AtB-AS/mittatb-app/assets/1777333/11461ca3-6e5e-4f09-bd01-c0e6b53a01ab)

Reply:
Yes, I think we should use 'Sykkel' as it's more generic

Reply:
Updated